nilearn/maskers/surface_labels_masker.py

```diff
@@ -30,59 +32,18 @@
 )
 from nilearn.image import mean_img
 from nilearn.maskers.base_masker import _BaseSurfaceMasker
-from nilearn.surface.surface import (
-    SurfaceImage,
-    at_least_2d,
-)
+from nilearn.surface.surface import SurfaceImage, at_least_2d, get_data
 from nilearn.surface.utils import check_polymesh_equal
 
 
-def _apply_surf_mask_on_labels(mask_data, labels_data, background_label=0):
-    """Apply mask to labels data.
-
-    Ensures that we only get the data back
-    according to the mask that was applied.
-    So if some labels were removed,
-    we will only get the data for the remaining labels,
-    the vertices that were masked out will be set to the background label.
-    """
-    labels_before_mask = {int(label) for label in np.unique(labels_data)}
-    labels_data[np.logical_not(mask_data.flatten())] = background_label
-    labels_after_mask = {int(label) for label in np.unique(labels_data)}
-    labels_diff = labels_before_mask - labels_after_mask
-    if labels_diff:
-        warnings.warn(
-            "After applying mask to the labels image, "
-            "the following labels were "
-            f"removed: {labels_diff}. "
-            f"Out of {len(labels_before_mask)} labels, the "
-            "masked labels image only contains "
-            f"{len(labels_after_mask)} labels "
-            "(including background).",
-            stacklevel=find_stack_level(),
-        )
-    labels = np.unique(labels_data)
-    labels = labels[labels != background_label]
-
-    return labels_data, labels
-
-
 def signals_to_surf_img_labels(
```

Comment:
I this is a private or public function ? If public, it should have full docstrings.

Reply:
na it's private
used in the parcellation module, hence why it has no leading underscore